### PR TITLE
adds v1 vm fixes back in

### PR DESCRIPTION
**Pull request by dandersonsw**
This adds back in the code needed for CF deployments upgraded from v1.10 to 2.x

### CODE REVIEW

File: hooks/blueprint [hunk 1]
```diff
@@ -348,18 +356,24 @@ for want in $GENESIS_REQUESTED_FEATURES; do
...
       manifest+=( "overlay/db/local-mysql-db.yml" )
       if want_feature '+override-db-names' ; then
-        manifest+=( \
-          "operations/db-override-names.yml" \
-          "operations/db-override-mysql-names.yml" \
-          "overlay/db/internal-overrides.yml" \
+      manifest+=( \
+        "operations/db-override-names.yml" \
+        "operations/db-override-mysql-names.yml" \
+        "overlay/db/internal-overrides.yml" \
```
Comment (dennisjbell) [inline hooks/blueprint:372]: Indentation changes for no reason

File: hooks/blueprint [hunk 2]
```diff
@@ -512,32 +551,58 @@ EOF
   )"
   if [[ -n "$instance_types" ]] ; then
     used=''
+
```
Comment (dennisjbell) [inline hooks/blueprint:554]: Unnecessary blank line

**Review by dennisjbell — CHANGES_REQUESTED** [review]
Please also include the changes for nats users (commits 9698794650c017bc574332aab9be1462d80f8a9e and 34afc91f6d0d77cb531349aa0a72a2f4a2f4ecfa), as well as the blobstore name fix (commit 1a68c0a75ebe720c29732241ad77dc03bbcbcf41).  These impact the v1 migration story.

**Comment by dandersonsw** [comment]
I've added the requested commits, and fixed the indentation and removed the unnecessary blank line.

**Review by dennisjbell — APPROVED** [review]

**Pull request merged**